tests/unit/services.py

```diff
@@ -62,6 +63,7 @@
     "aes_service",
     "jwt_service",
     "rsa_key_service",
+    "ltia_http_service",
```

Comment:
I think if you notice, all the services above `jstor` (which I think was added recently) are in alphabetical order here, which makes it a bit easier to scan when we have so many.

Reply:
Sorted this and the fixtures below :+1: